core/src/main/java/com/netflix/iceberg/BaseTableScan.java

```diff
@@ -24,6 +24,7 @@
 import com.google.common.cache.CacheBuilder;
 import com.google.common.cache.CacheLoader;
 import com.google.common.cache.LoadingCache;
+import com.google.common.collect.FluentIterable;
```

Comment:
How did I not know about this class before?

Reply:
I know I googled ``mapcat`` and landed on stackoverflow somehow, though now I cannot retrace my steps :)